host-configs/llnl/toss_4_x86_64_ib_cray/cce@12.0.3_clang_hip.cmake

```diff
@@ -0,0 +1,52 @@
+# Copyright (c) 2017-2021, Lawrence Livermore National Security, LLC and
```

Comment:
Since this compiler set is currently available on some machines, but not others (e.g. `rzwhamo` but not `rznevada`), I'm wondering if `rzwhamo` should be in the name of the file.

(It's in the comments and should work on `rznevada` in the near future, so perhaps that's enough).

Reply:
This comment is outdated -- `rznevada` now has `rocm@4.5`